packages/wrangler/src/d1/execute.tsx

```diff
@@ -246,7 +249,7 @@ async function executeLocally({
 		modules: true,
 		script: "",
 		d1Persist,
-		d1Databases: { DATABASE: id },
+		d1Databases: { DATABASE: binding },
```

Comment:
We need to do this so that databases created with `wrangler d1...` commands are still there when folks run `wrangler pages dev ... --d1=binding`

Reply:
Pages dev supports setting an ID in it's commands, e.g. `wrangler pages dev --d1 BINDING=db_id`. It feels like the right step here is to document that a little better (and won't be a concern when it receives config file support)